<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateNewProjectDialog</class>
 <widget class="QDialog" name="CreateNewProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>121</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="projectNameLabel">
       <property name="text">
        <string>Project name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="projectNameEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="projectFolderLabel">
       <property name="text">
        <string>Project folder</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="folderSelectButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="projectFileLabel">
       <property name="text">
        <string>Project file</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="projectFileEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="projectFolderEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>projectNameEdit</tabstop>
  <tabstop>projectFolderEdit</tabstop>
  <tabstop>folderSelectButton</tabstop>
  <tabstop>projectFileEdit</tabstop>
 </tabstops>
 <resources>
  <include location="../../../ugeneui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CreateNewProjectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CreateNewProjectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
